ledger-tool/src/bigtable.rs

```diff
@@ -498,3 +601,46 @@ pub fn bigtable_process_command(ledger_path: &Path, matches: &ArgMatches<'_>) {
         exit(1);
     });
 }
+
+fn missing_blocks(slots_std: &[Slot], slots_cmp: &[Slot]) -> Option<Vec<Slot>> {
+    if slots_cmp.is_empty() && !slots_std.is_empty() {
+        return Some(slots_std.to_owned());
+    } else if slots_cmp.is_empty() {
+        return None;
+    }
+    let set_cmp = HashSet::<&u64>::from_iter(slots_cmp.iter().clone());
```

Comment:
```suggestion
    let set_cmp: HashSet<_> = slots_cmp.iter().collect();
```
Slightly simpler way to write this.